app/helpers/blacklight_helper.rb

```diff
@@ -10,9 +10,11 @@ def document_partial_name(document, base_name = nil)
 
     display_type ||= document.display_type
 
+    display_type ||= document['eds_publication_type']
```

Comment:
Could we make that `eds_publication_type`?  The documents from EDS don't have a `format` field and we're not making use of that config anywhere for articles at the moment AFAIK.